FutureCollars kurs/src/main/java/com/futurecollars/lesson5/task1/TextVerifier.java

```diff
@@ -0,0 +1,7 @@
+package com.futurecollars.lesson5.task1;
+
+public class TextVerifier {
+    public static boolean containsCharacters(String text) {
```

Comment:
Dodatkowa klasa to trochę za dużo. Ta logika spokojnie mogła by być dodatkową metodą prywatną w klasie TextProcessor.